docs/src/main/asciidoc/property-source-config.adoc

```diff
@@ -5,18 +5,27 @@ an `application-profile.properties` or `application-profile.yaml` file that cont
 application or Spring Boot starters. You can override these properties by specifying system properties or environment
 variables.
 
+To enable this functionality you need to set `spring.config.import=kubernetes:` in your application's configuration properties.
+Currently you can not specify a ConfigMap or Secret to load using `spring.config.import`, by default Spring Cloud Kubernetes
```

Comment:
I remember last year when I initially looked at the config api with my limited knowledge of it, and this : "Currently you can not specify a ConfigMap or Secret to load" is what I found out too. I mean, I am sure there is a way to properly add this support, but I have not been able to pin point the needed things for this. Probably, for the initial push into this direction, it's a good thing, but in general, we are shaving off quite a lot of functionality from users. Still, may be in the future, we will have full support.

Reply:
> Still, may be in the future, we will have full support.

We are still maintining full support because all the properties and functionality that we had before still works, the only real change is when they are loaded and you need to have `spring.config.import=kubernetes:` to cause it to load (be activated).  

What I meant by this comment is that you can't do something like this
`spring.config.import=kubernetes:my-properties` and have a ConfigMap and/or secret with the name `my-properties` loaded, make sense?

Maybe we will support something like that in the future, that possibility is left open for sure, but right now with this implementation that would not work.